### ISSUE TITLE
CLI?

### ISSUE THREAD

**Issue by brianpipa** [deathg0d/geojson-mend#1]
Is there a way to use this on the commandline? geojson-precision works from a commandline - can you do the same to this so it's usable from CLI?

**Comment by deathg0d** [comment]
@brianpipa yes the CLI is missing, @lachana we should work on this when we have time.

**Comment by mvl22** [comment]
This would be very useful.
